shoov/modules/custom/shoov_ci_build/handlers/shoov_ci_build_trigger_button.inc

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * @file
+ * Contains shoov_ci_build_trigger_button.
+ */
+
+class shoov_ci_build_trigger_button extends views_handler_field_entity {
```

Comment:
lets improve the name to indicate what it is doing. Should probably not have the word button, since it's a link ;)